migration-engine/qe-setup/src/lib.rs

```diff
@@ -87,12 +87,12 @@ pub async fn teardown(prisma_schema: &str, db_schemas: &[&str]) -> ConnectorResu
 
 #[derive(Default)]
 struct LoggingHost {
-    printed: std::sync::Mutex<Vec<String>>,
+    printed: parking_lot::Mutex<Vec<String>>,
```

Comment:
Even better would be `tokio::sync::Mutex`, but I didn't want to change any migration engine apis.

Reply:
Standard `MutexGuard` is `!Send`, and is nasty to work with.